config/initializers/devise.rb

```diff
@@ -1,6 +1,7 @@
 # Use this hook to configure devise mailer, warden hooks and so forth.
 # Many of these configuration options can be set straight in your model.
 Devise.setup do |config|
+  config.secret_key = 'd3357bb9160a8c480b615bbe6a2f5421b067e2c785316e084a848f499d908260e56fa4953e936406dc7592f8ea617445639f8c873d63ef16285d0c67432014a6'
```

Comment:
don't ever push any secret key directly. It is security risk.

Read this http://www.justinweiss.com/blog/2015/02/24/how-to-configure-your-rails-app-to-ship/

Place all secret key in environment and use them